G/Glib/build_tarballs.jl

```diff
@@ -1,20 +1,39 @@
 using BinaryBuilder
 
 name = "Glib"
-version = v"2.59.0"
+version = v"2.62.0"
 
 # Collection of sources required to build Glib
 sources = [
     "https://ftp.gnome.org/pub/gnome/sources/glib/$(version.major).$(version.minor)/glib-$(version).tar.xz" =>
-    "664a5dee7307384bb074955f8e5891c7cecece349bbcc8a8311890dc185b428e",
+    "6c257205a0a343b662c9961a58bb4ba1f1e31c82f5c6b909ec741194abc3da10",
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/glib-*/
 mkdir build_glib && cd build_glib
-meson .. -Dman=false --cross-file="${MESON_TARGET_TOOLCHAIN}"
 
+SED_SCRIPT=()
+if [[ "${target}" == i686-linux-musl ]]; then
+    # We can't run executables for i686-linux-musl in the BB environment
+    SED_SCRIPT+=(-e "s/needs_exe_wrapper = false/needs_exe_wrapper = true/" "${MESON_TARGET_TOOLCHAIN}")
+elif [[ "${target}" == *-mingw* ]]; then
+    # Tell meson where to find libintl.h
+    SED_SCRIPT+=(-e "s?c_args = \[]?c_args = ['-I${prefix}/include']?")
```

Comment:
No clue why only when compiling for Windows I had to add `${prefix}/include` to the list of include paths in order to find `libintl.h`